app/src/main/java/com/framgia/englishconversation/screen/createPost/CreatePostViewModel.java

```diff
@@ -542,12 +563,13 @@ public void onRecordingAudioClick(String filePath, String fileName) {
                         record.setName(fileName);
                         initMedia(filePath);
                         addPostAudioRecord(record);
-                        initializeAudioPlayer();
+                        initializeAudioPlayer(mAudioFilePath, false,
```

Comment:
ERROR: 'method def' child have incorrect indentation level 24, expected level should be one of the following: 16, 20.